packages/csv-parser-discount-code/test/main.spec.js

```diff
@@ -0,0 +1,118 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
It looks like the `import/no-extraneous-dependencies` eslint  plugin does not work with parent package.json - it only checks the nearest one. I tried to move `streamtest` devDependency to `/packages/csv-parser-discount-code/package.json` and it stopped complaining. But when the devDep is in the main package.json, it complains with the message:
```eslint
  packages/csv-parser-discount-code/test/main.spec.js:1:1
  ✖    1:1   streamtest should be listed in the project's dependencies. Run npm i -S streamtest to add it  import/no-extraneous-dependencies
```
Also, here is the issue on the plugins github page:
https://github.com/benmosher/eslint-plugin-import/issues/458